src/main/groovy/liquibase/harness/compatibility/basic/BasicCompatibilityTestHelper.groovy

```diff
@@ -20,8 +20,8 @@ class BasicCompatibilityTestHelper {
     final static String baseChangelogPath = "liquibase/harness/compatibility/basic/changelogs"
     final static List supportedChangeLogFormats = ['xml', 'sql', 'json', 'yml', 'yaml'].asImmutable()
 
-    static boolean checkConnection(DatabaseConnection connection, String dbName) {
-        return connection.isClosed()||connection.getDatabaseProductName().toLowerCase().contains(dbName)
+    static boolean checkConnection(DatabaseConnection connection, String... dbNames) {
```

Comment:
rename both checkConnection methods to something like shouldOpenNewConnection(), needNewConnectionForDB(), etc. Something that is like question sentence that can be answered yes or no (true or false)